Gemfile

```diff
@@ -5,3 +5,5 @@ source "https://rubygems.org"
 # gem "rails"
```

Comment:
IMO, you should remove this comment after you generate the Gemfile for the first time.